### DISCUSSION TITLE
「藏寶圖」可預測南極 30 萬顆隕石藏身處

### DISCUSSION THREAD

**Discussion by Yao900108** [1101-GS4719-C/Python-Community-GS4719-C#530] [科技新知]
數千年來，太空隕石墜落並嵌入南極大陸冰原，隨著冰川緩慢流動。如果冰川遇到巨大障礙物，如橫貫南極山脈這種地形，冰層會上升，隕石被帶上地表。此外，乾燥的南極風逐漸侵蝕冰層也暴露出隕石。隨著更多冰層上升到地表，過程不斷重複，只要有足夠時間，就會推積大量隕石。
荷蘭Delft University of Technology研究人員利用人工智慧創建藏寶圖，可顯示南極洲最有可能找到隕石的地方，並指導研究人員去哪裡找。「經由分析了解，人造衛星對溫度、冰川速度、表面覆蓋和幾何形狀觀測可發揮預測隕石位置的功能，希望「藏寶圖」準確率能達80%。」領導研究的Veronica Tollenaar說。
我覺得這項技術讓我們對於大自然的理解更加透徹，也可以更深入的研究。
https://technews.tw/2022/02/14/treasure-map-predicts-the-hiding-places-of-300000-meteorites-across-antarctica/

**Comment by bloodtrail0511** [comment]
感謝分享，感覺除了應用在預測隕石位置，還有更多可能的應用如災害防治之類的

**Comment by Yuhsin000** [comment]
希望這個技術變成熟，日後去南極旅遊(?)時，找寶藏發大財比較容易XD

**Reply by Jay-722** [reply]
發大財+1

**Comment by Sazmafuji** [comment]
此技術的成熟，可能讓南極更多人去，到時候南極也變為觀光景點，使我們也可以去，雖然要思考一下環境問題

**Reply by chia-kai-chang** [reply]
老高：南極有很多秘密~

**Comment by ThomasMeng0515** [comment]
這對於想要藉由殞石來研究地質或是歷史的地質學家們會是一個很大的幫助

**Comment by sean1325418** [comment]
現在的全球暖化感覺會讓一堆隕石沉到大海裡面去

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，原來AI也可以用在這種地方。

**Comment by PoooYen** [comment]
隕石的蒐集也能幫助我們了解外太空的組成以及蒐集稀有元素

**Comment by yi-1111** [comment]
感覺之後各國研究學者都會趨之若鶩，奔向南極

**Comment by Jay-722** [comment]
南極有企鵝嗎，有的話會不會因為隕石變異成究極企鵝

**Reply by Yao900108** [reply]
會變成喬治企鵝

**Comment by HuanYuuuuuu** [comment]
即使對隕石的更加了解能對人類社會增加不少福祉，但同時還是要注意南極環境的保護

**Comment by yenwei12138** [comment]
這項技術既能讓人類更了解南極生態，又有錢途，希望能好好發展~

**Comment by Ben6uo** [comment]
謝謝分享!這項技術真的令我大開眼界

**Comment by cordydeng** [comment]
感覺是相當有趣的一項應用，不過如果真的找到鑽石了，那麼鑽石是屬於誰的呢?

**Comment by stan890425** [comment]
南極也有可愛的企鵝~~

**Comment by jcpan821** [comment]
如果能偵測海底沈船，就真的是藏寶圖了

**Comment by alexthebigpp** [comment]
這樣到時候在南極挖來挖去希望不會跑出甚麼恐怖的東西。

**Comment by Jimmy911218** [comment]
哇塞~感覺會不會在南極發生更酷炫的東西啊

**Comment by evanchou1204** [comment]
一般人好像不會去想到用AI來找埋在冰層裡的隕石，respect南極技術人員！

**Comment by Yunhans** [comment]
是時候去南極找氪星石了

**Comment by wenchi9023** [comment]
希望這項技術可以多加利用在其他地方，不然越多人去南極找隕石 冰川都要融化光了 : (

**Comment by Pietra1226** [comment]
其實變相的讓我更注重南極環境惡化的問題，希望能透過科技進步遏止氣候變遷快速惡化

**Comment by ABCDEFGHIJKLMNO12345678** [comment]
雖然AI運用在分析殞石對人類有很大的助益 但對環境的破壞也不可乎視
